httpServerLib.tcjs

```diff
@@ -15,13 +15,13 @@ tc.inclusionPaths = [
     pocoLoc + '\\Foundation\\include'
 ];
 
-tc.targetConfiguration = topTC.targetConfiguration || 'WinT.x64-VisualC++-15.0';
-tc.targetServicesLibrary = topTC.targetServicesLibrary || 'D:\\git\\rsarte-target-rts\\rsa_rt\\C++\\TargetRTS';
+tc.targetConfiguration = topTC.targetConfiguration || 'WinT.x64-VisualC++-17.0';
+tc.targetServicesLibrary = topTC.targetServicesLibrary;
 
 tc.unitName = 'UnitName_HttpLib';
 tc.copyrightText = `
 /*******************************************************************************
- * (c) Copyright HCL Technologies Ltd. 2019.  MIT Licensed!
+ * (c) Copyright HCL Technologies Ltd. 2024.  MIT Licensed!
```

Comment:
Copyright year should not be changed, but you can write 2019, 2024.

Reply:
corrected